kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/KieServerRouter.java

```diff
@@ -189,8 +239,18 @@ public void start(String host,
         pathHandler.addExactPath("/containers",
                                  new ContainersHttpHandler(notFoundHandler,
                                                            adminHandler));
-        pathHandler.addPrefixPath("/mgmt",
-                                  adminHandler);
+        HttpHandler mgmtHandler = adminHandler;
+        if (MANAGEMENT_SECURED) {
+            IdentityManager idm = getIdentityService();
+            mgmtHandler = new AuthenticationCallHandler(mgmtHandler);
```

Comment:
I think it may be better not to reuse mgmtHandler reference, but rather assign objects to different variables. It will increase code readability.